android/KMAPro/kMAPro/src/main/AndroidManifest.xml

```diff
@@ -103,6 +103,9 @@
         <data
           android:mimeType="application/octet-stream"
           android:scheme="content" />
+        <data
+          android:mimeType="application/x-keyman-package"
+          android:scheme="content" />
```

Comment:
Should we add a application/zip mime type? Given .kmp will be sniffed as a zip file.

```suggestion
        <data
          android:mimeType="application/x-keyman-package"
          android:scheme="content" />
        <data
          android:mimeType="application/zip"
          android:scheme="content" />
```

Reply:
hmm, I tried adding `application/zip` but then we'll need additional code handling in the engine to expect `.zip` files instead of strictly `.kmp` files.

Reply:
Feature for another day then